app/models/container_build.rb

```diff
@@ -9,7 +9,7 @@ class ContainerBuild < ApplicationRecord
            :as         => :resource,
            :inverse_of => :resource
 
-  has_many :container_build_pods
+  has_many :container_build_pods, :dependent => :destroy
```

Comment:
seeing only 10s of these, perhaps they're managed elsewhere, such as delete evens.

Reply:
`container_build_pods` are a top level managed object which should be deleted during refresh